lib_comfyui/async_comfyui_loader.py

```diff
@@ -23,9 +24,10 @@ def start_comfyui(comfyui_path):
     webui_resources_sharing.share_webui_folder_paths(folder_paths)
     custom_extension_injector.register_webui_extensions()
     print('[sd-webui-comfyui]', f'Launching ComfyUI with arguments: {" ".join(sys.argv[1:])}')
-    runpy.run_path(os.path.join(comfyui_path, "main.py"), {}, '__main__')
+    runpy.run_path(os.path.join(comfyui_path, 'main.py'), {}, '__main__')
 
 
 @dataclasses.dataclass
 class WebuiProcessModule(types.ModuleType):
     state_dict_queue: multiprocessing.Queue
+    opts_outdirs_queue: multiprocessing.Queue

```

Comment:
Can we replace this with a `@property` `opts`? Or it's a bad idea

Reply:
what do you mean by that? Can you show what it would look like?

Reply:
```python
@dataclasses.dataclass
class WebuiProcessModule(types.ModuleType):
    state_dict_queue: multiprocessing.Queue
    opts_queue: multiprocessing.Queue

    @property
    def opts(self):
        return self.opts_queue.get()
```

Reply:
yes, we could do it with state_dict_queue as well, but I'd rather not because it hides the fact that getting the value can be heavy computationaly

Reply:
Can we drop the @property but still introduce the function then? I think the queue is an implementation detail of how we fetch the options.

Reply:
Maybe call it `fetch_opts` or something that indicates you don't want to be fetching them continuously in a loop.